metadata-ingestion/examples/library/dataset_field_add_tag_patch.py

```diff
@@ -0,0 +1,24 @@
+from datahub.emitter.mce_builder import make_dataset_urn
+from datahub.emitter.rest_emitter import DataHubRestEmitter
+from datahub.metadata.schema_classes import TagAssociationClass
+from datahub.specific.dataset import DatasetPatchBuilder
+
+# Create DataHub Client
+rest_emitter = DataHubRestEmitter(gms_server="http://localhost:8080")
+
+# Create Dataset URN
+dataset_urn = make_dataset_urn(
+    platform="snowflake", name="fct_users_created", env="PROD"
+)
+
+# Create Dataset Patch to Add + Remove Tag for 'profile_id' column
+patch_builder = DatasetPatchBuilder(dataset_urn)
+patch_builder.for_field("profile_id").add_tag(
+    TagAssociationClass("urn:li:tag:tag-to-add-id")
```

Comment:
let's use `make_tag_urn` here

Reply:
Not sure sure about this. I think people should not generally be interesting in understanding the IDs and URN structure. I feel like URNs should be the default for logical metadata like Tags, Glossary Terms, and Domains

Reply:
imo this is more straightforward and requires less understanding of urn structure

Users do need to know that this is an urn, but don't need to understand the structure/format of an urn

```suggestion
    TagAssociationClass(make_tag_urn("tag-to-add-id"))
```